src/server/error.ts

```diff
@@ -0,0 +1,10 @@
+import { bold, red } from "./deps.ts";
+
+export function printError(message: string) {
+  console.error(red(`${bold("error")}: ${message}`));
+}
+
+export function error(message: string): never {
+  printError(message);
+  Deno.exit(1);
+}
```

Comment:
This has been duplicated from https://github.com/denoland/fresh/blob/main/src/dev/error.ts